DNN Platform/DotNetNuke.DependencyInjection/README.md

```diff
@@ -0,0 +1,56 @@
+# DotNetNuke.DependencyInjection project
+
+This project exposes `IDnnStartup`, which any component can implement in order
+to register services with DNN's dependency injection container.  It also exposes
+an extension method on the `Assembly` class, `SafeGetTypes`, which gets all of
+the `Type` objects from the assembly while handling potential load errors.
+
+This project targets .NET Standard 2.0, meaning that it is compatible with both
+.NET Framework 4.7.2 and .NET Core 2.0.  Any interfaces defined in this project

```

Comment:
.NET Standard 2.0 is compatible with .NET Framework 4.6.1 and newer. It is also compatible with .NET Core 2.0 and newer, this includes .NET 5. It may be useful to link to the official Microsoft documentation on what versions can be used in the many .NETs

https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Reply:
Yeah, I said .NET Framework 4.7.2 b/c that's what DNN Platform is targeting, and b/c it's not recommended to target .NET Standard with 4.6.x (footnote 2 on the table in that doc you linked).  Maybe just remove the versions altogether:

```suggestion
.NET Framework and .NET Core.  Any interfaces defined in this project
```